src/app/dataset-view/dataset-view-header/dataset-view-header.model.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright Kamu Data, Inc. and contributors. All rights reserved.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the LICENSE file.
+ */
+
+import { SearchAdditionalHeaderButtonOptions } from "src/app/common/components/search-additional-buttons/search-additional-buttons.interface";
+import { searchAdditionalButtonsEnum } from "src/app/search/search.interface";
+
+export const searchAdditionalButtonsDescriptors: SearchAdditionalHeaderButtonOptions = {
+    [searchAdditionalButtonsEnum.Starred]: {
+        id: "search.starred",
+        textButton: searchAdditionalButtonsEnum.Starred,
+        counter: 2,
+        iconSvgPath:
+            "M8 .25a.75.75 0 01.673.418l1.882 3.815 4.21.612a.75.75 0 01.416 1.279l-3.046 2.97.719 4.192a.75.75 0 01-1.088.791L8 12.347l-3.766 1.98a.75.75 0 01-1.088-.79l.72-4.194L.818 6.374a.75.75 0 01.416-1.28l4.21-.611L7.327.668A.75.75 0 018 .25z",
+        svgClass: "octicon octicon-star-fill starred-button-icon d-inline-block mr-2",
```

Comment:
Let's have a normal SVG icon files like we have everywhere, why embedding those into HTML...

Reply:
Done